src/containers/NFT/NFTHeader/Details.tsx

```diff
@@ -15,7 +15,16 @@ interface Props {
 }
 
 export const Details = ({ data }: Props) => {
-  const { minted, domain, NFTTaxon, uri, transferFee, owner, isBurned } = data
+  const {
+    minted,
+    domain,
+    NFTTaxon: nftTaxon,
```

Comment:
perhaps it should also be changed wherever it gets `data` from? Not important right now